packages/ui-library/src/elements/toast/docs/close.md

```diff
@@ -0,0 +1 @@
+The `Toast.Close` subcomponent allows to render a button that closes the toast. It is useful when you want to give the user the ability to close the toast manually.
```

Comment:
Maybe I would refactor this doc as follow: 

The `Toast.Close` subcomponent renders a Toast with a button to close the toast. This feature is useful when you want to provide the user with the option to manually close the toast notification.

Reply:
Technically the `Toast.Close` it is a subcomponent of the `Toast` element, so it renders just the close button. I'd rephrase it as: 
The `Toast.Close subcomponent` renders a button to close the toast. This feature is useful when you want to provide the user with the option to manually close the toast notification.